src/loggly.tracker.js

```diff
@@ -155,7 +164,11 @@
                 //creating an asynchronous XMLHttpRequest
                 var xmlHttp = new XMLHttpRequest();
                 xmlHttp.open('POST', this.inputUrl, true); //true for asynchronous request
-                xmlHttp.setRequestHeader('Content-Type', 'text/plain');
+                if (tracker.useUtfEncoding === true) {
```

Comment:
`useUtfEncoding` is already a boolean so no need to check with `true, false or undefined`.

`data.useUtfEncoding` for `true` and `!data.useUtfEncoding` for `false or undefined` will be sufficient here.

Reply:
Got it, thanks @psamit .